regtests/0155_user_sock/usock.adb

```diff
@@ -119,9 +119,11 @@ package body USock is
       Port          : Natural;
       Host          : String := "";
       Reuse_Address : Boolean := False;
+      IPv6_Only     : Boolean := False;
```

Comment:
This is an incompatible change and so should be documented in the corresponding readme.txt section, right?

Reply:
Right!

Reply:
added record: